lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -352,27 +445,35 @@ sub fetch_db_meta {
 # TODO  parameter for species
 # TODO  use assemblyID
 # Assembly not taken to account, assembly of REST machine
-# TODO Report by individul datasets - only 1 currently
 sub variant_exists {
-  my ($self, $reference_name, $start, $end, $ref_allele, 
-           $alt_allele, $incl_ds_response, $dataset) = @_;
+  my ($self, $reference_name, $start, $start_max, $end, $end_min, $ref_allele, 
```

Comment:
params galore, consider refactoring to a hash params

Reply:
This method is getting bigger and more complicated. I'll do as suggested below and PR separately in the future but I'll let it that this for now if that's ok.